Home page and logged out page styling

jtkabenni opened the pull request:
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Updates to home page and logged out page styling according to high fidelity mockups. Also changed name to clevercart and implemented colors. 

## Related Issue

closes #121 
closes #107 
closes #105 

## Acceptance Criteria

<!-- Include AC from the Github issue -->

## Type of Changes

Styling update

## Updates

### Before

Mobile:
<img width="500" alt="Screenshot 2024-04-04 at 1 40 23 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/c31b8418-ece2-489e-87a9-e3c78667ca09">
<img width="500" alt="Screenshot 2024-04-04 at 1 40 18 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/ed93b1f8-9a19-4a1e-af28-cb4a505243c6">

web:
<img width="1440" alt="Screenshot 2024-04-04 at 1 40 11 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/4e95d385-9cdd-4397-9a7e-80c2893099f5">
<img width="1440" alt="Screenshot 2024-04-04 at 1 40 03 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/5c590a3f-9ba7-493c-a23a-86e8e5d78f4d">

### After
Mobile: 
<img width="499" alt="Screenshot 2024-04-04 at 1 38 27 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/b4f70662-9b46-4e41-85fe-570500c64ee3">
<img width="499" alt="Screenshot 2024-04-04 at 1 38 15 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/a9b77000-240f-4109-ae9a-7e5fc04dc33d">

Web: 
<img width="1440" alt="Screenshot 2024-04-04 at 1 39 18 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/1de7bc84-b219-43d9-a4a3-e900f68edf57">
<img width="1440" alt="Screenshot 2024-04-04 at 1 39 28 PM" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/46732901/13d2964f-ae61-4b11-a464-c73ae240de59">


## Testing Steps / QA Criteria

- View logged out page
- View mobile logged out page
- Log in 
- View home page
- View mobile home page

jtkabenni committed:
updates to home and logged out page layout

flbarfield commented:
It's looking so goooood!!!!

I'm noticing some wonk with the input boxes and text on the login screen, though, at certain widths

<img width="526" alt="Ex1" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/14006060/b79b86b8-cc25-4a8a-9839-26811d8cd828">

<img width="497" alt="ex2" src="https://github.com/the-collab-lab/tcl-67-smart-shopping-list/assets/14006060/97503b85-30dd-407b-85f3-1f4f56ebacd2">

bdanzer commented on src/index.css:
I mentioned this in our [channel](https://the-collab-lab.slack.com/archives/C06FWHTGYCV/p1712265955006869). Since we haven't done a lot of CSS in this cohort there hasn't been an opportunity to bring this up. But components and css should be isolated together. Having global css files with changes can cause all kinds of issues and generally speaking most companies are not using files like these anymore.

aberger3647 reviewed (changes requested):
We're missing the black borders on mobile screen size, homepage view

jtkabenni committed:
adding css file for LoggedOut component

jtkabenni committed:
adding borders and logged out image

jtkabenni committed:
fixing merge errors:

jtkabenni committed:
fixing merge errors css

jtkabenni committed:
removing extra welcome message

jtkabenni commented:
Hey @aberger3647  and @flbarfield , I made the following changes:
- created separate css files for AddListForm, LoggedOut, SelectListForm, and removed those styles from index.css
- changed the sizing of all the headers in web (you will have to adjust the headers list view according to what looks right)
- added an image for the logged out page image. @aberger3647 if you want to try to create it in css please feel free to.
- updated our app name to clevercart
- added borders where needed and removed it in some other places so they wouldn't double up

jtkabenni committed:
fixing selected list dropdown value

aberger3647 reviewed (approved):
Looks awesome. Thanks for updating the border and breaking out the CSS files
